### DISCUSSION TITLE
How to split PRs on major version?

### DISCUSSION THREAD

**Discussion by Robula** [renovatebot/renovate#12765] [Request Help]
In Renvote PR's how would I specify different rules for a particular dependency? For example, I have a monorepo with some packages that use Webpack v4 due to compatibility reasons and others that use Webpack v5.

### EXAMPLE:
![image](https://user-images.githubusercontent.com/15246425/142662346-70ac1611-5d42-4f78-8477-9c3f25d2f371.png)

I'd like to receive minor updates to both Webpack 4 and 5 without upgrading all versions to the latest v5 as per the automatic PR. Is this possible?

Here is my config: 
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":dependencyDashboard",
    ":prHourlyLimitNone",
    ":prConcurrentLimitNone",
    ":automergeMinor",
    ":automergeTypes"
  ]
}
```

Many thanks :)

**Comment by viceice** [comment]
You should disable major update for webpack

**Reply by Robula** [reply]
Thanks, but I haven't set `separateMajorMinor` to false anywhere, yet the default is true.

**Comment by viceice** [comment]
Or use dependency Dashboard approval
